daffodil-lib/src/main/scala/org/apache/daffodil/oolag/OOLAG.scala

```diff
@@ -308,7 +308,7 @@ object OOLAG extends Logging {
      * Unconditionally, evaluate the LV arg in order to insure all checks for this
      * object are performed.
      */
-    private def requiredEvaluationsAlways(lv: OOLAGValueBase) : Unit = {
+    private def requiredEvaluationsAlways(lv: OOLAGValueBase) (): Unit = {
```

Comment:
A few of the changes in this file look incorrect--looks like theres an extra set of parens. Is this a accidental find/replace due to the space before the colon?